quartz/components/styles/breadcrumbs.scss

```diff
@@ -0,0 +1,22 @@
+.breadcrumb-container {
```

Comment:
we should make it so that this takes up 100% width and overflow-x has scroll (have you tried what the behaviour is like on small screens?)

Reply:
yes, i added `flex-wrap: wrap`, so every breadcrumb-element just starts a new line, if it doesn't have enough space. works on desktop and mobile:

<img width="510" alt="image" src="https://github.com/jackyzha0/quartz/assets/31989404/f58306c8-93c0-441b-abe7-e4048d916a5d">

<img width="820" alt="image" src="https://github.com/jackyzha0/quartz/assets/31989404/c7ecfc52-c117-4419-baec-4b5ab6982748">